mbs_results/staging/data_cleaning.py

```diff
@@ -316,8 +317,10 @@ def run_live_or_frozen(
         )
 
     if state == "frozen":
-
-        df.loc[df[error_marker].isin(error_values), target] = np.nan
+        df["frozen_error"] = df.apply(
```

Comment:
Nice use of lambda function to simplify copying over the value from the target column :)